Lib/imghdr.py

```diff
@@ -36,7 +36,7 @@ def what(file, h=None):
 
 def test_jpeg(h, f):
     """JPEG data in JFIF or Exif format"""
-    if h[6:10] in (b'JFIF', b'Exif'):
+    if h[:2] == b'\xff\xd8':
```

Comment:
And what’s the behaviour with the existing files?

Reply:
It returns True only for `python.jpg` and False otherwise.
This is the same as the previous code.